backend.native/compiler/ir/backend.native/src/org/jetbrains/kotlin/backend/konan/objcexport/StubRenderer.kt

```diff
@@ -18,6 +24,8 @@ object StubRenderer {
                 }
                 +"*/"
             }
+            this.descriptor?.findKDoc()?.let { +it.render() }
```

Comment:
Can it be merged with `this.comment` above?

Reply:
(I guess we would avoid this for now, see https://github.com/JetBrains/kotlin-native/pull/4441#discussion_r558416537)